SpatialGDK/Source/SpatialGDK/Private/Interop/GlobalStateManager.cpp

```diff
@@ -93,6 +107,33 @@ void UGlobalStateManager::ApplyAcceptingPlayersUpdate(bool bAcceptingPlayersUpda
 	}
 }
 
+void UGlobalStateManager::SendShutdownMultiProcessRequest(bool bValue)
+{
+	Internal_SendShutdownMultiProcessRequest();
+}
+
+void UGlobalStateManager::Internal_SendShutdownMultiProcessRequest()
+{
+	FURL DummyURL;
+	Worker_CommandRequest CommandRequest = {};
+	CommandRequest.component_id = SpatialConstants::GSM_SHUTDOWN_COMPONENT_ID;
+	CommandRequest.schema_type = Schema_CreateCommandRequest(SpatialConstants::GSM_SHUTDOWN_COMPONENT_ID, 1);
+	Schema_Object* RequestObject = Schema_GetCommandRequestObject(CommandRequest.schema_type);
+	AddStringToSchema(RequestObject, 1, DummyURL.ToString(true));
+
+	NetDriver->Connection->SendCommandRequest(GlobalStateManagerEntityId, &CommandRequest, 1);
+}
+
+void UGlobalStateManager::ReceiveShutdownMultiProcessRequest()
+{
+	if (NetDriver && NetDriver->GetNetMode() == NM_DedicatedServer)
+	{
+		UE_LOG(LogGlobalStateManager, Warning, TEXT("Received shutdown multi-process request."));
+		SetAcceptingPlayers(false);
```

Comment:
Can you verify that this update is making its way into the runtime?

Reply:
yes, i did.